package.json

```diff
@@ -11,7 +11,9 @@
     "prettier": "prettier",
     "typescript": "tsc",
     "prepublishOnly": "npm run build",
-    "test": "jest --config jest.config.json"
+    "test": "jest --config jest.config.json",
+    "test:watch": "jest --watchAll --config jest.config.json",
```

Comment:
fyi, these can be accomplished without adding scripts the package.json: `npm test -- --watch.`

I agree though that this makes it easier for people to know how to watch the tests. I actually prefer doing `--watch` instead of `--watchAll` so that only the files I've changed will have their tests run.

Reply:
agreed